packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -19,38 +21,44 @@ export class Arc extends React.Component<IArcProps, IArcState> {
 
   public state: {} = {};
 
+  private _classNames: IProcessedStyleSet<IArcStyles>;
   private currentRef = React.createRef<SVGPathElement>();
 
   public static getDerivedStateFromProps(nextProps: Readonly<IArcProps>): Partial<IArcState> | null {
     _updateChart(nextProps);
     return null;
   }
 
+  public constructor(props: IArcProps) {
+    super(props);
+  }
+
   public updateChart(newProps: IArcProps): void {
     _updateChart(newProps);
   }
 
   public render(): JSX.Element {
     const { arc, href, focusedArcId } = this.props;
     const getClassNames = classNamesFunction<IArcProps, IArcStyles>();
-    const classNames = getClassNames(getStyles, {
+    this._classNames = getClassNames(getStyles, {
```

Comment:
why add this?

Reply:
added this when the truncation function was in this file, changed it back